SPECS/ldns/ldns.spec

```diff
@@ -32,48 +32,39 @@
 
 Summary:        Low-level DNS(SEC) library with API
 Name:           ldns
-Version:        1.7.0
-Release:        32%{?dist}
-License:        BSD
+Version:        1.8.3
+Release:        1%{?dist}
+License:        BSD-3-Clause
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
+Distribution:   Azure Linux
 Group:          System Environment/Libraries
 Url:            http://www.nlnetlabs.nl/%{name}/
 Source0:        http://www.nlnetlabs.nl/downloads/%{name}/%{name}-%{version}.tar.gz
-Patch1:         ldns-1.7.0-multilib.patch
-Patch2:         ldns-1.7.0-parse-limit.patch
-Patch3:         ldns-1.7.0-realloc.patch
-Patch4:         ldns-1.7.0-Update-for-SWIG-4.patch
 
-# Only needed for builds from svn snapshot
-%if 0%{snapshot}
```

Comment:
If we're removing all uses of `%{snapshot}`, I think we should also remove this line at the top of the spec:
```
%{?!snapshot:         %global snapshot        0}
```